### PR TITLE
[new][author] Add d3.chart, version 0.2.1 

### PR DESCRIPTION
**Pull request by iros**
References #5435

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity : Watch 61, Star 651, Fork 77
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.

### CODE REVIEW

**Comment by Amomo** [comment]
Hi @iros , to make commits look clear in cdnjs, please squash them into only one commit.
Thank you~ :-)

**Comment by iros** [comment]
Just did! Tests running =) 
Thanks @Amomo.

**Comment by iros** [comment]
Let me know if there's anything I need to do to get this added =) Much thanks for your help!

File: ajax/libs/d3.chart/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "main": "d3.chart",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/misoproject/d3.chart.git",
+    "basePath": "",
+    "files": [
+      "d3.chart.js",
+      "d3.chart.min.js"
+    ]
```
Comment (PeterDaveHello) [inline ajax/libs/d3.chart/package.json]: Forgot to add map file here?

Comment (Amomo) [inline ajax/libs/d3.chart/package.json]: Hi @iros , could you also add `d3.chart.min.map` in "files"?

**Comment by iros** [comment]
Done. Thanks so much for catching that! Sorry about missing it.

**Comment by Amomo** [comment]
@iros could you rebase the commit to match our latest master branch?
Thank for your help. :)

**Comment by iros** [comment]
Done @Amomo.

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is ok, could you take a look?
Thank you~

**Comment by PeterDaveHello** [comment]
GJ @iros

**Comment by iros** [comment]
Thanks all! I'll make sure to get this right the first time next time heh.

**Comment by Amomo** [comment]
😄:+1:

**Comment by PeterDaveHello** [comment]
@iros You are welcome to send new PRs. :+1:

**Pull request merged**